lib/cookie.js

```diff
@@ -58,7 +58,7 @@ var CONTROL_CHARS = /[\x00-\x1F]/;
 // (see: https://github.com/ChromiumWebApps/chromium/blob/b3d3b4da8bb94c1b2e061600df106d590fda3620/net/cookies/parsed_cookie.cc#L60)
 // '=' and ';' are attribute/values separators
 // (see: https://github.com/ChromiumWebApps/chromium/blob/b3d3b4da8bb94c1b2e061600df106d590fda3620/net/cookies/parsed_cookie.cc#L64)
-var COOKIE_PAIR = /^(([^=;]+))\s*=\s*([^\n\r\0]*)/;
+var COOKIE_PAIR = /^(([^=;]+))\s{0,512}=\s*([^\n\r\0]*)/;
```

Comment:
It would be nice to leave a comment for future generations about why we need this limitation.

Reply:
And we need a regression test for this.

Reply:
It looks like `trim()` is called on the captured key/val anyways, so why bother with the `\s*` altogether?  It's already in the capture group `[^=;]`.

Reply:
Also, note that `LOOSE_COOKIE_PAIR` seems to have the same problematic pattern.